src/main/java/org/commonjava/indy/service/tracking/change/FoloTrackingListener.java

```diff
@@ -67,7 +67,7 @@ public void handleFileAccessEvent( final FileEvent event )
          * We should avoid the tracker if it sends request through generic proxy and the target is
          * indy instance,
          */
-        if ( originPath != null && originPath.contains( "api/folo/track" ) )
+        if ( originPath != null && originPath.contains( "api/tracking/track" ) )
```

Comment:
the originPath should be still folo one, which points to monolith to request the artifact.

Reply:
updated